server/tinode.conf

```diff
@@ -190,7 +190,14 @@
 				// See https://github.com/go-sql-driver/mysql#dsn-data-source-name for syntax.
 				"dsn": "root@tcp(localhost)/tinode?parseTime=true&collation=utf8mb4_unicode_ci",
```

Comment:
You are going to hard-fail on this config. Please provide a default working config.

Reply:
Done.